hawk/src/main/java/com/orhanobut/hawk/HawkBuilder.java

```diff
@@ -196,4 +202,23 @@ public static Storage newSqliteStorage(Context context) {
 
     void onFail(Exception e);
   }
+    public Observable<Boolean> buildRx() {
+        return Observable.defer(new Func0<Observable<Boolean>>() {
```

Comment:
Since Rx is optional, would be great if you add a check, I've added this in Hawk.java but that can be moved to utils and used here as well.
```java
private static void checkRx() {
    if (!Utils.hasRxJavaOnClasspath()) {
      throw new NoClassDefFoundError("RxJava is not on classpath, " +
          "make sure that you have it in your dependencies");
    }
  }
```